### PR TITLE
Fix #1737: Correct output directory for multi-module Maven build

### LINKED ISSUE #1737: Wrong Jib output directory in Maven multi-module build
### Description
For a multi-module Maven build the Jib output directory is wrong.

#### Given the following Maven multi-module setup:
```
.
|— a
|— b
```

#### Actual behavior; The Jib Docker tarballs are created in ./target:
```
.
|— a
|— b
|— target
   |—docker
     |— a.tar
     |— b.tar         
```
#### Expected behavior; , The Jib Docker tarballs are created in ./a/target and ./b/target
```
.
|— a
   |— target
      |—docker
        |— a.tar

|— b
   |— target
      |—docker
        |— b.tar         
```

### Info
* f-m-p version : 4.3.0 (and HEAD of master 17.10.2019)
* Maven version (`mvn -v`) : 3.5.3

### PR DESCRIPTION
**Pull request by erikgb**
This is the simplest fix I could come up with now. My time to spend on this is sadly limited at the moment, but I think a lot should be improved with the Jib feature in f-m-p.

### CODE REVIEW

File: core/src/main/java/io/fabric8/maven/core/service/kubernetes/JibBuildService.java [hunk 1]
```diff
@@ -65,7 +64,7 @@ public void build(ImageConfiguration imageConfiguration) {
            jibBuildConfiguration = JibBuildServiceUtil.getJibBuildConfiguration(config, buildImageConfiguration, fullName, log);
            JibBuildServiceUtil.buildImage(jibBuildConfiguration, log);
        } catch (Exception ex) {
-           throw new UnsupportedOperationException();
```
Comment (erikgb) [inline core/src/main/java/io/fabric8/maven/core/service/kubernetes/JibBuildService.java]: Re-throwing exceptions without the cause should be **illegal**! 😩

**Comment by rohanKanojia** [comment]
let me merge this for now, then I can start working on #1738 on top of this.

**Comment by erikgb** [comment]
Thanks @rohanKanojia! I really really really want to use the new Jib feature, but the quality must definitely be improved! Already looking forward to f-m-p 4.3.1. 😃

File: core/src/main/java/io/fabric8/maven/core/service/kubernetes/JibBuildService.java (hunk 1 above)
Comment (dev-gaur) [reply core/src/main/java/io/fabric8/maven/core/service/kubernetes/JibBuildService.java]: aah, really sorry! my bad

File: core/src/main/java/io/fabric8/maven/core/util/JibBuildServiceUtil.java [hunk 2]
```diff
@@ -43,8 +43,8 @@
 import io.fabric8.maven.docker.access.AuthConfig;
 import io.fabric8.maven.docker.config.BuildImageConfiguration;
 import io.fabric8.maven.docker.service.RegistryService;
+import io.fabric8.maven.docker.util.EnvUtil;
```
Comment (dev-gaur) [inline core/src/main/java/io/fabric8/maven/core/util/JibBuildServiceUtil.java]: @rohanKanojia We should copy the prepareAbsoluteOutputDirPath method in fmp from dmp, while reflecting this PR in jshift.

Comment (rohanKanojia) [reply core/src/main/java/io/fabric8/maven/core/util/JibBuildServiceUtil.java]: not sure what you mean, We already have this method in jkube/jshift https://github.com/jkubeio/jkube/blob/master/jkube-kit/common/src/main/java/io/jkube/kit/common/util/EnvUtil.java#L413

**Pull request merged**